### ISSUE TITLE
Objects and arrays are being parsed as strings for attributes

### ISSUE THREAD

**Issue by eddyloewen** [webtides/luna-js#65]
I think I found the issue:

```git
diff --git a/node_modules/@webtides/luna-js/src/framework/engine/element-renderer.js b/node_modules/@webtides/luna-js/src/framework/engine/element-renderer.js
index 58891f3..2a25b69 100644
--- a/node_modules/@webtides/luna-js/src/framework/engine/element-renderer.js
+++ b/node_modules/@webtides/luna-js/src/framework/engine/element-renderer.js
@@ -28,7 +28,8 @@ export default class ElementRenderer {
             // First we define the static properties as they have the least priority
             ...component.element.staticProperties,
             // Then the attributes, as they are somewhat dynamic
-            ...attributes,
+            // TODO: this seems to be a problem because the attributes alreay have been parsed in buildElement() and these are the original ones... :(
+            // ...attributes,
             // The highest priority have dynamic properties.
             ...dynamicProperties,
         };

```

**Comment by lukas-schardt** [comment]
Solved by #61

**Comment by lukas-schardt** [comment]
Closed by #67
